src/containerapp/azext_containerapp/_utils.py

```diff
@@ -726,3 +726,22 @@ def parse_build_env_vars(env_list):
         })
 
     return env_var_def
+
+
+def get_subscription_cloud_name(cli_ctx):
+    from azure.cli.core._profile import (Profile, _ENVIRONMENT_NAME)
+    subscription_id = get_subscription_id(cli_ctx)
+
+    return Profile(cli_ctx=cli_ctx).get_subscription(subscription=subscription_id)[_ENVIRONMENT_NAME]
```

Comment:
why do you need to get cloud information from subscription? No cloud info in cli_ctx?

Reply:
use https://github.com/Azure/azure-cli/blob/f98ba8062f08bf5b7460260c56916bd3074bf9e5/src/azure-cli-core/azure/cli/core/cloud.py#L575

Reply:
Fixed. Use `cli_ctx.cloud.name`